docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -218,9 +220,9 @@ Configuration
 Further Reading
 ---------------
 
```

Comment:
Please use the text I provided for the see also. It is consistent with formatting of this section in the other docs.